integration-cli/docker_cli_build_test.go

```diff
@@ -4920,50 +3903,23 @@ func (s *DockerSuite) TestBuildFromMixedcaseDockerfile(c *check.C) {
 	testRequires(c, UnixCli) // Dockerfile overwrites dockerfile on windows
 	testRequires(c, DaemonIsLinux)
 
-	ctx, err := fakeContext(`FROM busybox
```

Comment:
This test was weird too… The way it was written, there was both `Dockerfile` and `dockerfile` in the build context, with the same content… thus it was not testing what it  was supposed to test...